default.yml

```diff
@@ -127,3 +127,6 @@ Style/Documentation:
 # value in using "" for all strings irrespective of whether string interpolation is used
 Style/StringLiterals:
   EnforcedStyle: double_quotes
+
+Style/HashSyntax:
+  Enabled: false
```

Comment:
I think we need to enable this but with `EnforcedShorthandSyntax: either` so that the newer `foo:, bar:` style works but is optional.

Reply:
ah good point, I'll change that now.